lang/lang_en_ro.txt

```diff
@@ -528,7 +528,7 @@
 
 #MSG_LOAD_FILAMENT c=17
 "Load filament"
-"Incarca fil."
+"Incarca filamentul"
```

Comment:
Nu incape. D-aia l-am lasat asa.

Reply:
![20220115_123110.jpg](https://user-images.githubusercontent.com/56076331/149618669-44f627aa-2800-4e4e-9383-0bf060ebc18b.jpg)

Reply:
Aceeasi problema ca la Scoate filamentul. E folosit si la multimaterial mesajul unde pune codul un numar dupa mesaj si nu incape pe ecran acolo

Reply:
> OK modific inapoi, o sa mai incerc odata cu mmu.

Reply:
La partea asta spunei ![20220115_134222.jpg](https://user-images.githubusercontent.com/56076331/149620626-d5a69b32-37ad-4e33-901b-8c780fc6564f.jpg)

Reply:
Da. Load filament, eject filament si cut filament au numar dupa ele, asa ca trebuie sa fie mai scurte putin

Reply:
@leptun Am Modificat deja in "Incarca filament" numa poza era facuta inainte sa o modific :) si nu am mai dat push la modificarile noi  inca.